bottlerocket-settings-models/modeled-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bottlerocket-modeled-types"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
there are crates in this project that depend on this, so we'll need to bump this version in those too

Reply:
Nice catch! Faster than the CI 🤠